src/community/pom.xml

```diff
@@ -64,8 +64,9 @@
           <descriptor>release/ext-gwc-distributed.xml</descriptor>  
           <descriptor>release/ext-geofence-server.xml</descriptor>
           <descriptor>release/ext-gwc-s3.xml</descriptor>	
-	  <descriptor>release/ext-gdal-wcs.xml</descriptor>
-	  <descriptor>release/ext-gdal-wps.xml</descriptor>
+	  	  <descriptor>release/ext-gdal-wcs.xml</descriptor>
+	      <descriptor>release/ext-gdal-wps.xml</descriptor>
+	      <descriptor>release/ext-wps-jdbc.xml</descriptor>
```

Comment:
What happened to indentation here? I can see the original was already using tabs, which is not nice, but it looks like it got worse or something